### ISSUE TITLE
PU-psnr bug

### ISSUE THREAD

**Issue by jgerster0** [gfxdisp/ColorVideoVDP#47]
The pu-psnr seems to have a bug. Shouldnt it be: mse += torch.mean( (T_enc - R_enc)**2, dim=(1,2,3,4) )

	def predict_video_source(self, vid_source, frame_padding="replicate"):
        _, _, N_frames = vid_source.get_video_size()
        batch_sz = vid_source.get_batch_size()

        mse = torch.zeros((batch_sz), device=self.device)
        for ff in range(N_frames):
            T = vid_source.get_test_frame(ff, device=self.device, colorspace=self.metric_colorspace)
            R = vid_source.get_reference_frame(ff, device=self.device, colorspace=self.metric_colorspace)

            # Apply PU
            T_enc = self.pu.encode(T)
            R_enc = self.pu.encode(R)

            mse += torch.mean( (T - R)**2, dim=(1,2,3,4) )
        
        psnr = 20*torch.log10( self.max_I/torch.sqrt(mse/N_frames) ) 

        return psnr, None

**Comment by mantiuk** [comment]
Yes, of course, it should be R_enc and T_enc. Fixed in #fb0b6df.
